### PR TITLE
Add chartControl to WinformsControlsTest project

### PR DESCRIPTION
**Pull request by LeafShi1**
## Proposed changes

- Add chartControl to WinformsControlsTest project

## Customer Impact

-  Developers can debug ChartControl related code in the Runtime repo

## Regression? 

- No

## Risk

- Minimal

### Before
No ChartControl button in the WinformsControlsTest project

### After

AfterChange

## Test methodology 

- manually
 

## Test environment(s) 

- .net 9.0.0-alpha.1.23566.1

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/10330)

### CODE REVIEW

File: src/System.Windows.Forms/tests/IntegrationTests/WinformsControlsTest/ChartControl.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            // Fill series data for chart2
+            chart2.ChartAreas[0].AxisX.LabelStyle.IntervalOffset = 1;
+            chart2.ChartAreas[0].AxisX.LabelStyle.IntervalOffsetType = DateTimeIntervalType.Days;
+            chart2.ChartAreas[0].AxisX.LabelStyle.Interval = 2;
+            chart2.ChartAreas[0].AxisX.LabelStyle.IntervalType = DateTimeIntervalType.Days;
+
+            // Fill series data for chart3
+            double[] yValues = { 32.4, 56.9, 89.7, 80.5, 59.3, 33.8, 78.8, 44.6, 76.4, 68.9 };
+            chart3.Series["DataSeries"].Points.DataBindY(yValues);
+            chart3.Series["ErrorBar"]["ErrorBarSeries"] = "DataSeries"; // Link error bar series with data series
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/tests/IntegrationTests/WinformsControlsTest/ChartControl.cs:35]: Please put the comment on a dedicated line above the corresponding line of code.

**Review by Tanya-Solyanik — COMMENTED** [review]
Looks good, I added one nit only.

**Review by Tanya-Solyanik — APPROVED** [review]

**Review by Tanya-Solyanik — APPROVED** [review]

**Comment by Tanya-Solyanik** [comment]
@LeafShi1 - Consider breaking the test application main UI into 3 columns when you have time.

**Comment by Nora-Zhou01** [comment]
Verified this on .NET 9.0.100-alpha.1.23564.26 + private dlls(build from main branch of WinForms repo), chartControl exists in the WinformsControlsTest project.

https://github.com/dotnet/winforms/assets/104609169/53e58085-82d5-4490-95b5-85e35cc6b87b

**Pull request merged**